tests/scenario_tests_async/test_events.py

```diff
@@ -129,6 +129,16 @@ async def test_reaction_added(self):
         await asyncio.sleep(1)  # wait a bit after auto ack()
         assert self.mock_received_requests["/chat.postMessage"] == 1
 
+    @pytest.mark.asyncio
+    async def test_stable_auto_ack(self):
+        app = AsyncApp(client=self.web_client, signing_secret=self.signing_secret,)
```

Comment:
Trailing comma intentional here?

Reply:
No strong opinion. Either is fine. So far, most code consistently has a trailing comma.

Reply:
I saw the other `AsyncApp` signature of `test_stable_auto_ack` didn't feature it, so wasn't sure